examples/image_i2c.rs

```diff
@@ -30,8 +30,7 @@ extern crate stm32f1xx_hal as hal;
 
 use cortex_m_rt::ExceptionFrame;
 use cortex_m_rt::{entry, exception};
-use embedded_graphics::image::Image1BPP;
-use embedded_graphics::prelude::*;
+use embedded_graphics::{image::ImageLE, prelude::*};
```

Comment:
`ImageLE` is is already included in the prelude.

Reply:
Ah nuts, I don't think any of the `Image*` exports should be in the prelude tbh. None of the primitives or fonts are so that's a mistake on my part for not spotting that in review. I'm going to open an issue in embedded-graphics to remove it.

Reply:
For reference: #169.